pnpm-lock.yaml

```diff
@@ -1,247 +1,173 @@
-lockfileVersion: '6.0'
+lockfileVersion: 5.4
```

Comment:
please update your pnpm version to 8.x and revert this lockfile change

Reply:
pnpm version is alraedy 8.6.0 but if i run `pnpm i` this file will be generated again with version 5.4

Reply:
are you sure? that sounds very odd

Reply:
https://github.com/aptos-labs/explorer/assets/121921928/175cd260-9a72-4402-a7e4-adddffa5ab8e

Reply:
you forgot the most important part in the video - `pnpm --version` :-P